jobs-builder/jobs/include/ci_entrypoint.sh.inc

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Red Hat, Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Wrapper shell script to call kata-containers/tests/.ci/ci_entry_point.sh
+# with expected environment variables exported.
+#
+
+set -e
+set -x
```

Comment:
Nit: Might be worth adding the usual header:

```bash
set -o errexit
set -o nounset
set -o pipefail

[ -n "$BASH_VERSION" ] && set -o errtrace
[ -n "${DEBUG:-}" ] && set -o xtrace
```

Reply:
This script is just a thin wrapper to https://raw.githubusercontent.com/kata-containers/tests/main/.ci/ci_entry_point.sh , which in turn adds all those flags :)